UnityProject/Assets/Scripts/Chat/Chat.Process.cs

```diff
@@ -83,6 +89,24 @@ private static (string, ChatModifier) ProcessMessage(ConnectedPlayer sentByPlaye
 			message = message.Substring(1);
 			chatModifiers |= ChatModifier.Whisper;
 		}
+		// Whisper alias
+		else if (message.StartsWith("/w "))
+		{
+			message = message.Substring(3);
+			chatModifiers |= ChatModifier.Whisper;
+		}
+		// Sing
+		else if (message.StartsWith("%"))
+		{
+			message = message.Substring(1);
+			chatModifiers |= ChatModifier.Sing;
+		}
+		// Sing alias
+		else if (message.StartsWith("/s" ))
```

Comment:
I think there's a space missing here. Should this be
`else if (message.StartsWith(" /s" ))`
instead?